### DISCUSSION TITLE
404 Error

### DISCUSSION THREAD

**Discussion by Dragonslayer42069** [dani-garcia/vaultwarden#3239] [Q&A]
Hi,

after installation I get 404 error and following log

`Feb 10 12:14:32 vaultwarden[24457]: /--------------------------------------------------------------------\
Feb 10 12:14:32 vaultwarden[24457]: |                        Starting Vaultwarden                        |
Feb 10 12:14:32 vaultwarden[24457]: |                           Version 1.27.0                           |
Feb 10 12:14:32 vaultwarden[24457]: |--------------------------------------------------------------------|
Feb 10 12:14:32 vaultwarden[24457]: | This is an *unofficial* Bitwarden implementation, DO NOT use the   |
Feb 10 12:14:32 vaultwarden[24457]: | official channels to report bugs/features, regardless of client.   |
Feb 10 12:14:32 vaultwarden[24457]: | Send usage/configuration questions or feature requests to:         |
Feb 10 12:14:32 vaultwarden[24457]: |   https://vaultwarden.discourse.group/                             |
Feb 10 12:14:32 vaultwarden[24457]: | Report suspected bugs/issues in the software itself at:            |
Feb 10 12:14:32 vaultwarden[24457]: |   https://github.com/dani-garcia/vaultwarden/issues/new            |
Feb 10 12:14:32 vaultwarden[24457]: \--------------------------------------------------------------------/
Feb 10 12:14:32 vaultwarden[24457]: [WARNING] The following environment variables are being overriden by the config file,
Feb 10 12:14:32 vaultwarden[24457]: [WARNING] please use the admin panel to make changes to them:
Feb 10 12:14:32 vaultwarden[24457]: [WARNING] DOMAIN, SIGNUPS_ALLOWED, ADMIN_TOKEN, SMTP_HOST, SMTP_PORT, SMTP_FROM, SMTP_FROM_NAME
Feb 10 12:14:32 vaultwarden[24457]: [2023-02-10 12:14:32.753][vaultwarden::api::notifications][INFO] Starting WebSockets server on 127.0.0.1:3012
Feb 10 12:14:32 vaultwarden[24457]: [2023-02-10 12:14:32.758][start][INFO] Rocket has launched from http://127.0.0.1:8095
Feb 10 12:15:10 vaultwarden[24457]: [2023-02-10 12:15:10.801][_][ERROR] No matching routes for GET /vaultwarden/admin text/html.
Feb 10 12:15:10 vaultwarden[24457]: [2023-02-10 12:15:10.802][_][WARN] No 404 catcher registered. Using Rocket default.
Feb 10 12:15:13 vaultwarden[24457]: [2023-02-10 12:15:13.927][_][ERROR] No matching routes for GET /vaultwarden/api text/html.
Feb 10 12:15:13 vaultwarden[24457]: [2023-02-10 12:15:13.927][_][WARN] No 404 catcher registered. Using Rocket default.
Feb 10 12:15:14 vaultwarden[24457]: [2023-02-10 12:15:14.885][_][ERROR] No matching routes for GET /vaultwarden/ text/html.
Feb 10 12:15:14 vaultwarden[24457]: [2023-02-10 12:15:14.885][_][WARN] No 404 catcher registered. Using Rocket default.`

**Comment by RealOrangeOne** [comment]
Looking at the URLs, you're trying to visit `/vaultwarden/admin`, which won't work. If you want to serve vaultwarden on a subpath (ie `/vaultwarden`), you'll need to strip that prefix before passing it to vaultwarden.

**Reply by BlackDex** [reply]
Actually, you need to include this in the `DOMAIN` variable, else stuff will not work at all, like some 2FA, and mails will point to the wrong URL.

**Reply by Dragonslayer42069** [reply]
Where do I have to do it?
I installed it via yunohost.

**Reply by RealOrangeOne** [reply]
However you set environment variables through yunohost
